configs/scripts/program/build.sh

```
@@ -1,142 +1,41 @@
 OUTPUT="./programs/.bin"
 # saves external programs binaries to the output directory
-"${SCRIPT_DIR}/dump.sh" "${OUTPUT}"
+source ${SCRIPT_DIR}/dump.sh ${OUTPUT}
 
 # Dump programs currently on devnet only
-"${SCRIPT_DIR}/dump_devnet.sh" "${OUTPUT}"
+source ${SCRIPT_DIR}/dump_devnet.sh ${OUTPUT}
 
 # go to parent folder
-cd "$(dirname "$(dirname "$(dirname "${SCRIPT_DIR}")")")"
+cd $(dirname $(dirname $(dirname ${SCRIPT_DIR})))
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)